src/js/index.js

```diff
@@ -0,0 +1,115 @@
+import '../css/style.css';
+import tasks from './tasks.js';
+import taskManager from './taskManager.js';
+
+const markupAllLists = () => {
+  let allLists = '';
+  tasks.todoData.forEach((data, index) => {
+    allLists += `<li class="todo-item">
+                    <div class="todo-entry" id=entry_${index}>
+                      <input type="checkbox" ${data.completed ? 'checked' : ''} class="status" title="Check!">
+                      <input type="text" class="todo" value="${data.description}">
```

Comment:
-Use the map() method instead of forEach() to create an array of todo items markup, and then join the array to create the final markup. This eliminates the need to declare an empty string variable and manually concatenate each todo item markup.
javascript

```javascript
const markupAllLists = () => {
  const allLists = tasks.todoData.map((data, index) => markupTodoItem(data, index));
  return allLists.join('');
};
```